pkg/linter/messages.yaml

```diff
@@ -8259,12 +8259,11 @@ LinterLintCode:
       stable: "2.0"
     categories: [style]
     hasPublishedDocs: false
-    deprecatedDetails: |-
+    documentation: |-
```

Comment:
This one should still be `deprecatedDetails`. `documentation` is reserved for a format consistent with the analyzer's diagnostic docs.